plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/OpenShiftCommonImages.java

```diff
@@ -39,5 +39,11 @@
 	public static final ImageDescriptor SYSTEM_PROCESS = repo.create("systemprocess.gif"); //$NON-NLS-1$ 
 	public static final Image SYSTEM_PROCESS_IMG = repo.getImage("systemprocess.gif"); //$NON-NLS-1$ 
 
+	public static final ImageDescriptor WARNING = repo.create("pficon-warning.png"); //$NON-NLS-1$ 
```

Comment:
@adietish removed warning icon in preference to eclipse error.  kept the ok as I dont see an equivalent and the info icon doesnt properly represent success IMO